static/components/AppAround.js

```diff
@@ -124,8 +128,8 @@
 	        return _react2.default.createElement(_PoiCard2.default, {
 	          key: key,
 	          details: _this3.state.pois[key],
-	          userLat: _this3.state.userLocation[0],
-	          userLng: _this3.state.userLocation[1]
+	          userLng: _this3.state.userLocation[0],
```

Comment:
`_this`, `_this2`, `_this3` are confusing.
try arrow functions whenever you can and comment when you need to do weird things.